C++/2. Variables.md

```diff
@@ -2,59 +2,78 @@
 
 ## Start
 
-Variables are containers which hold a specific something. They are words which contain something in them, which you can call or take by just typing out the word. Variables in C++ are created by first typing out their type. Each variable has a type, and there are a few types of variables:
+Variables are containers which can hold values of a specific type. They are words which contain something in them, which you can call or take by just typing out the word. Variables in C++ are created by first typing out their type. Each variable has a type, and there are an infinite number of types that you can create. The language provides a few primitives types, which can be used to build larger, or more complicated types, here are a few:
 
+- `bool`
+- `char`
+- `short`
+- `unsigned`
 - `int`
+- `long`
 - `float`
+- `long long`
 - `double`
-- `std::string`
-- `char`
-- `bool`
 
 ## Explanation
 
-The `int` variable type means an integer, which means a whole number. \
-The `float` variable type means a single-precision decimal value. \
-The `double` variable type means a double-precision decimal value. \
-The `std::string` variable type means a string or a / some word(s). Strings are encased in double quotes "". \
-The `char` variable type means a character. `char` stands for character, which means one number or letter. `char`s are encased in single quotes ''. \
-The `bool` variable type means a `true` or `false` variable. The values it can be are those itself.
+The `bool` type means a boolean, which can only be `true` or `false`. The values it can be are those itself.
+The `char` type means a (ASCII) character. `char` stands for character, which means one number or letter. `char`s are encased in single quotes ''.
+The `short` type means a short integer. It can usually hold integers as smaller than `int`, and may take less space.
+The `int` variable type means an integer, which means a whole number.
+The `unsigned` type means an unsigned integer. Unsigned refers to how it lacks a sign, as in a `+` or `-` sign. For example, `4` is a valid value of `unsigned`, whereas `-4` is not. It can hold (slightly) higher positive values, yet no negatives.
+The `long` variable type means a long integer, it can hold up to values as large as or larger than `int`.
+The `float` variable type means a single-precision decimal value.
+The `long long` variable type means a longer integer, it can hold up to values as large as or larger than `long`.
+The `double` variable type means a double-precision decimal value.
```

Comment:
Hey put the meanings in the same order as you stated them above

Reply:
Oh, I thought that I did order them?

I'll fix that soon.